locales/ES_es.json

```diff
@@ -177,34 +176,27 @@
 				"github": "https://github.com/Kinnay"
 			},
 			{
-				"name": "NinStar",
-				"caption": "Icons for the Mii Editor and Juxt reactions",
-				"picture": "https://github.com/ninstar.png",
-				"github": "https://github.com/ninstar"
-			},
```

Comment:
NinStar is missing in your locale

Reply:
Ready, I will to fix it. Sorry for the inconvenients